components/button.tsx

```diff
@@ -0,0 +1,26 @@
+import classnames from 'classnames'
+import type { FC } from 'react'
+
+type ButtonProps = {
+  className?: string
+}
+
+export const ButtonPrimary: FC<ButtonProps> = ({ className, ...props }) => (
+  <button
```

Comment:
I guess I should, you know, actually make these buttons do something when you click them